### PR TITLE
build: release 2.3.0

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**
/deploy renku=000-tests-ui-2_0-next #persist

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
 # Changes
 
+
+# [2.3.0](https://github.com/SwissDataScienceCenter/renku-ui/compare/2.2.0...2.2.0) (2022-05-23)
+
+
+### Bug Fixes
+
+* force listing projects on datasets ([#1842](https://github.com/SwissDataScienceCenter/renku-ui/issues/1842), [#1843](https://github.com/SwissDataScienceCenter/renku-ui/issues/1843))
```
Comment (ciyer) [inline CHANGELOG.md:9]:
```suggestion
* correct listing of projects on datasets ([#1842](https://github.com/SwissDataScienceCenter/renku-ui/issues/1842), [#1843](https://github.com/SwissDataScienceCenter/renku-ui/issues/1843))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+  * Dataset image ([#1828](https://github.com/SwissDataScienceCenter/renku-ui/issues/1828), [#1829](https://github.com/SwissDataScienceCenter/renku-ui/issues/1829))
+  * Project forms ([#1802](https://github.com/SwissDataScienceCenter/renku-ui/issues/1802), [#1770](https://github.com/SwissDataScienceCenter/renku-ui/issues/1770), [#1736](https://github.com/SwissDataScienceCenter/renku-ui/issues/1736))
+  * Visibility ([#1747](https://github.com/SwissDataScienceCenter/renku-ui/issues/1747), [#1733](https://github.com/SwissDataScienceCenter/renku-ui/issues/1733))
+  * Progress ([#1761](https://github.com/SwissDataScienceCenter/renku-ui/issues/1761), [#1735](https://github.com/SwissDataScienceCenter/renku-ui/issues/1735), [#1774](https://github.com/SwissDataScienceCenter/renku-ui/issues/1774), [#1775](https://github.com/SwissDataScienceCenter/renku-ui/issues/1775))
+  * Templates ([#1734](https://github.com/SwissDataScienceCenter/renku-ui/issues/1734), [#1760](https://github.com/SwissDataScienceCenter/renku-ui/issues/1760))
```
Comment (ciyer) [inline CHANGELOG.md:26]:
```suggestion
* ux improvements to project creation ([#1734](https://github.com/SwissDataScienceCenter/renku-ui/issues/1734), [#1760](https://github.com/SwissDataScienceCenter/renku-ui/issues/1760), [#1736](https://github.com/SwissDataScienceCenter/renku-ui/issues/1736), [#1770](https://github.com/SwissDataScienceCenter/renku-ui/issues/1770), [#1802](https://github.com/SwissDataScienceCenter/renku-ui/issues/1802))
* ux improvements to dataset creation ([#1747](https://github.com/SwissDataScienceCenter/renku-ui/issues/1747), [#1733](https://github.com/SwissDataScienceCenter/renku-ui/issues/1733), [#1738](https://github.com/SwissDataScienceCenter/renku-ui/issues/1738), [#1827](https://github.com/SwissDataScienceCenter/renku-ui/issues/1827), [#1739](https://github.com/SwissDataScienceCenter/renku-ui/issues/1739), [#1815](https://github.com/SwissDataScienceCenter/renku-ui/issues/1815), [#1828](https://github.com/SwissDataScienceCenter/renku-ui/issues/1828), [#1829](https://github.com/SwissDataScienceCenter/renku-ui/issues/1829))
* ux improvements to progress indicator ([#1761](https://github.com/SwissDataScienceCenter/renku-ui/issues/1761), [#1735](https://github.com/SwissDataScienceCenter/renku-ui/issues/1735), [#1774](https://github.com/SwissDataScienceCenter/renku-ui/issues/1774), [#1775](https://github.com/SwissDataScienceCenter/renku-ui/issues/1775))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
 # Changes
 
+
+# [2.3.0](https://github.com/SwissDataScienceCenter/renku-ui/compare/2.2.0...2.2.0) (2022-05-23)
```
Comment (ciyer) [inline CHANGELOG.md:4]:
```suggestion
## [2.3.0](https://github.com/SwissDataScienceCenter/renku-ui/compare/2.2.0...2.3.0) (2022-05-23)
```

**Review by ciyer — CHANGES_REQUESTED** [review]
A few proposals for the text of the change log.

**Comment by lorenzo-cavazzi** [comment]
Thanks for the suggestions! Your way of presenting the UX changes is definitely better :smile:

**Review by ciyer — APPROVED** [review]
🎉

**Pull request merged**